PythonRegression/tests/features/steps/stitching_steps.py

```diff
@@ -0,0 +1,82 @@
+from aloe import step
+from util.test_logic import api_test_logic as tests
+from util.transaction_utils import transaction_logic as transactions
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+config = {}
+responses = {}
+
+side_tangle_address = b'SIDE9TANGLE9999999999999999999999999999999999999999999999999999999999999999999999'
+stitching_addresss = b'STITCHING9TRANSACTIONS99999999999999999999999999999999999999999999999999999999999'
+referencing_address = b'REFERENCES9STITCHING9TRANSACTION9999999999999999999999999999999999999999999999999'
```

Comment:
I am now working to try to obtain some side tangle Txs and a database on which to find and stitch them. In my opinion this should not be merged until we can make a 1.5.2 node fail on a stitching transaction. We need to prove that this indeed was fixed between 1.5.2 and 1.5.3. The result should be that if you run this set of tests on a latest `dev` this test won't fail, but it will if you run it on a revision prior to 1.5.3.